src/init-es.js

```diff
@@ -31,7 +31,11 @@ const initES = async () => {
     const body = { mappings: {} }
     body.mappings[config.get('ES.ES_TYPE')] = {
       properties: {
-        id: { type: 'keyword' }
+        id: { type: 'keyword' },
+        memberHandle: { 
```

Comment:
Does this change mean we have to do anything to the existing Dynamo DB? I don't see any change to the model. Just making sure

Reply:
We don't need to change any thing with Dynamo DB, because search is making in Elasticsearch.